config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :dish, only: [:index, :show, :new, :create, :edit, :update]
+  resources :chef, only: [:index, :show]
 end
```

Comment:
You have a lot of unimplemented routes in this file. Remember that we don't want to create routes that we do not have implementation for in our controller. 

I also noticed that you do not have a `ChefsController` but this could also be because you ran out of time.